src/components/BorrowerProfile/LendCta.vue

```diff
@@ -590,6 +594,9 @@ export default {
 					this.$kvTrackEvent('Borrower profile', 'Complete loan', 'click-amount-left-cta', this.loanId, this.selectedOption);
 				}
 			}).catch(e => {
+				if (e?.message !== INVALID_BASKET_ERROR) {
+					this.$kvTrackEvent('Lending', 'Add-to-Basket', 'Failed to add loan. Please try again.');
```

Comment:
`add-to-basket` though

Reply:
Quick follow-up, should the category be some BP related, like @roger-in-kiva mentioned?

Reply:
Lowercase all.

Reply:
I wouldn't change existing analytics, in case we have reports using to them.